### PR TITLE
Mheavin snow 596599 readme fix

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**
### Description
I deleted the README.rst file and replaced it with a README.md file.

### Checklist
- [ ] Code compiles correctly
- [ ] Run ``make fmt`` to fix inconsistent formats
- [ ] Run ``make lint`` to get lint errors and fix all of them
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Go
+
+The latest driver requires the [Go language](https://golang.org/) 1.17 or higher. The supported operating systems are Linux, Mac OS, and Windows, but you may run the driver on other platforms if the Go language works correctly on those platforms.
+
+
+# Installation
+
+Get Gosnowflake source code, if not installed.
+
+```sh
```
Comment (sfc-gh-mgilkey) [inline README.md:28]: Are the presence and placement of "sh" deliberate? 

My knee-jerk reaction is that it looks as though we're trying to use /bin/sh to run a "go" command, which seems a bit odd, and also seems inconsistent with the fact that the "sh" is on a different line from the "go". 

Or maybe this is MarkDown notation that I don't recognize (I don't know MarkDown).

I know I'm asking questions, not giving answers.  :-(

File: README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+In the following example, the program ``select1.go`` is built and installed in ``$GOPATH/bin`` and can be run from the command line:
+
+```
+SNOWFLAKE_TEST_ACCOUNT=<your_account> \
+SNOWFLAKE_TEST_USER=<your_user> \
+SNOWFLAKE_TEST_PASSWORD=<your_password> \
+select1
+Congrats! You have successfully run SELECT 1 with Snowflake DB!
```
Comment (sfc-gh-mgilkey) [inline README.md:53]: You might want to separate what the reader should type (line 49-52) from the output that the reader will see after the command executes (line 53).

**Review by sfc-gh-mgilkey — APPROVED** [review]
This looks fine to me (other than my few comments in-line), but I do not know MarkDown and could have missed some problems.

After this is visible on the web site, please send me the URL of the published version so that I can review it again looking at it the way that the customers will see it.

BTW: Thank you for cleaning this up. :-)

File: README.md (hunk 2 above)
Comment (sfc-gh-mheavin) [reply README.md:53]: Not a way to do it markdown, as it does not support formatting within code blocks.

 I inserted a blank line to give a little separation.

File: README.md (hunk 1 above)
Comment (sfc-gh-mheavin) [reply README.md:28]: Those were shell commands, which is why I used "sh". It doesn't really make a difference in this code blocks, though, so I just removed them.

**Comment by sfc-gh-mheavin** [comment]
> This looks fine to me (other than my few comments in-line), but I do not know MarkDown and could have missed some problems.
> 
> After this is visible on the web site, please send me the URL of the published version so that I can review it again looking at it the way that the customers will see it.
> 
> BTW: Thank you for cleaning this up. :-)

From what I've read, I can't actually see what the go help looks like until I merge the branch, which kicks of the doc build. Before I merge, I want to do some more research. It seems silly to keep creating and merging branches just to see how Go generates the doc output.

**Comment by sfc-gh-mgilkey** [comment]
> From what I've read, I can't actually see what the go help looks like until I merge the branch, which kicks of the doc build. Before I merge, I want to do some more research. It seems silly to keep creating and merging branches just to see how Go generates the doc output.

I had the same experience. :-(

I think we should be able to copy the file to our laptops, then process the MarkDown with whatever the MarkDown "compiler" is, but I don't have a copy of the MarkDown "compiler".

File: README.md (hunk 2 above)
Comment (sfc-gh-mgilkey) [reply README.md:53]: That's fine for now.
But for future reference, remember that you could split this into two code blocks.

**Pull request merged**